src/XMLexport.cpp

```diff
@@ -475,8 +475,8 @@ void XMLexport::writeHost(Host* pHost, pugi::xml_node mudletPackage)
             static_cast<int>(pHost->mBlankLineBehaviour));
     host.append_attribute("NetworkPacketTimeout") = pHost->mTelnet.getPostingTimeout();
     if (int mode = static_cast<int>(pHost->getControlCharacterMode()); mode) {
```

Comment:
With what I now know about `enum`s and `QFlags` I'd have done it a bit different and instead stuck to using proper values, i.e.:
```suggestion
    if (ControlCharacterMode mode = pHost->getControlCharacterMode(); mode != ControlCharacterMode::AsIs) {
```
though a further revision to line 481 may also be needed...